### DISCUSSION TITLE
Error handling response: TypeError: Cannot read properties of null (reading '1')

### DISCUSSION THREAD

**Discussion by l1mindr** [vercel/next.js#44835] [Help] [answered]
Hi
I'm creating new nextjs project 
But when I run the project in the phone frame, this log is printed in the console
```js 
Error handling response: TypeError: Cannot read properties of null (reading '1')
    at chrome-extension://pgjjikdiikihdfpoppgaidccahalehjh/webspeed.js:9:73298
    at chrome-extension://pgjjikdiikihdfpoppgaidccahalehjh/webspeed.js:9:73485
```

**Comment by icyJoseph** [comment, marked as answer]
Have you tried running in Incognito mode? The logs seem to indicate that the issue is contained within an extension you have installed, called, webspeed (?)

**Reply by l1mindr** [reply]
That's right, thanks for your help my friend

**Reply by icyJoseph** [reply]
Nice. Don't forget to mark as answered :)
